packages/main/src/TableRow.ts

```diff
@@ -128,6 +152,14 @@ class TableRow extends TableRowBase {
 		return Promise.resolve();
 	}
 
+	_ondragstart() {
```

Comment:
why this is not done by the DragRegistry should every component do the same?

Reply:
Sometimes the target element is not the one, which should be dragged.

This happens in the TabContainer, where custom logic is applied: DragRegistry.setDraggedElement((e.target as TabInStrip).realTabReference);. To override the default behavior, it registers selfManagedDragAreas.

Also for the Table it is not clear where to unsubscribe:
`DragRegistry.subscribe(this._table); // TODO: Where unsubscribe?`

We believe the code is now simpler and clearer.

The table DnD logic is moved to TableDragAndDrop